packages/block-editor/src/components/block-list/block-selection-button.js

```diff
@@ -31,21 +31,23 @@ function BlockSelectionButton( {
 	moverDirection,
 	...props
 } ) {
-	const selected = useSelect(
+	const { index, name, attributes } = useSelect(
 		( select ) => {
 			const {
 				__unstableGetBlockWithoutInnerBlocks,
 				getBlockIndex,
 			} = select( 'core/block-editor' );
-			const index = getBlockIndex( clientId, rootClientId );
-			const { name, attributes } = __unstableGetBlockWithoutInnerBlocks(
+			const { _name, _attributes } = __unstableGetBlockWithoutInnerBlocks(
```

Comment:
we can just keep block undestructed here or something to avoid the eslint error, underscore for variables are weird (I know it's not the only place we use them)